src/testdir/test_visual.vim

```diff
@@ -1630,4 +1630,40 @@ func Test_visual_substitute_visual()
   bwipe!
 endfunc
 
+func Test_visual_getregion()
+  new
+  call setline(1, ['one', 'two', 'three'])
+
+  call cursor(1, 1)
+  execute "normal!" "\<ESC>vl"
```

Comment:
Is it possible to use feedkeys() here Instead of "normal!"?  Does it end the visual selection?